reana_client/cli/workflow.py

```diff
@@ -1204,16 +1183,15 @@ def workflow_diff(
 
     def print_color_diff(lines):
         for line in lines:
-            line_color = None
+            msg_type = None
             if line[0] == "@":
-                line_color = "cyan"
+                msg_type = "info"
             elif line[0] == "-":
-                line_color = "red"
+                msg_type = "error"
             elif line[0] == "+":
-                line_color = "green"
-            click.secho(line, fg=line_color)
+                msg_type = "success"
+            display_message(line, msg_type=msg_type)
```

Comment:
*suggestion*: conceptually it is not really good that differences in lines are marked with "error", "success", etc. instead of colors. I would leave colors here as an exception. In addition, msg_types can introduce unwanted formatting and formatting is very important for checking differences.

Reply:
Not sure what you mean by _conceptually_.

In any case, if you would like to use _colors_ instead of the high-level _msg_type_ argument, the `display_message` function would need to be replaced by `click.secho` here (which defeats the purpose of centralising everything, but we could make an exception.).

Want to leave `click.secho`?

Reply:
Let me check how `display_message` will affect lines formatting. I will post here the findings.

Reply:
What I meant by "conceptually": the difference is showing that some lines were removed (`-`), this is not an `error` or `success`, this is just stating "this line is removed", but the code that prints those removed lines is saying`msg_type="error"` which states this is an error.

Reply:
@Sinclert let's keep the original version of difference cmd. As you can see from screenshots that @mvidalgarcia posted a new version looks overwhelming. Thank you.

Reply:
Reverted in https://github.com/reanahub/reana-client/pull/553/commits/cbe2b47c260ae496cfbddedb33233dc5f79d770d